tests/Tokens/compLikeTest.js

```diff
@@ -0,0 +1,22 @@
+const {
+  makeCToken,
+} = require('../Utils/Compound');
+
+
+describe('CCompLikeDelegate', function () {
+  describe("_delegateCompLikeTo", () => {
+    it("does not delegate if not the admin", async () => {
+      const [root, a1] = saddle.accounts;
+      const cToken = await makeCToken({kind: 'ccomp'});
+      await expect(send(cToken, '_delegateCompLikeTo', [a1], {from: a1})).rejects.toRevert('revert only the admin may set the comp-like delegate');
```

Comment:
This works due to the fallback in `CErc20Delegator`, right?

Reply:
Yeah it delegates everything